media-sound/milkytracker/patches/milkytracker-1.01.00.patchset

```diff
@@ -36,7 +36,7 @@ index 39e0412..2931ecf 100644
  SubDirHdrs - ;
  
 -LINKLIBS on MilkyTracker = -lbe -lgame -lmedia -lmidi2 -ltextencoding -ltracker ;
-+LINKLIBS on MilkyTracker = -lbe -lgame -lmedia -lmidi2 -ltextencoding -ltracker -lstdc++.r4 ;
```

Comment:
Why is this needed when you only tested on x86_64?

Reply:
Nah, i didn't changed anything here, just applied the patchset, added the missing file, extracted the patches, and it got changed. Why: it is unclear to me. I can however change them back.